addons/survey/models/survey_question.py

```diff
@@ -472,12 +472,22 @@ def _validate_date(self, answer):
         return {}
```

Comment:
Commit message missing task reference (even if not strictly related, we always add a task ID for later reference).